charts/sysdig/Chart.yaml

```diff
@@ -15,4 +15,4 @@ name: sysdig
 sources:
```

Comment:
U can revent the changes in the `sysdig` chart since it's deprecated and we should not maintain it anymore